packages/core/src/codecs/json-codec.ts

```diff
@@ -24,7 +24,7 @@ export default class JsonCodec implements ContentCodec {
     private subMediaType: string;
 
     constructor(subMediaType?: string) {
-        if (!subMediaType) {
+        if (subMediaType == null) {
             this.subMediaType = ContentSerdes.DEFAULT; // 'application/json'
         } else {
             this.subMediaType = subMediaType;
```

Comment:
Could be simplified to

```ts
this.subMediaType = subMediaType ?? ContentSerdes.DEFAULT;
```

Reply:
https://github.com/eclipse-thingweb/node-wot/pull/1096/commits/994e1a7861af07dd058cae4c4e410c60051ab647